### PR TITLE
Migrate documentation to mkdocs

### PR DESCRIPTION
**Pull request by m-kuhn**
Migration

```sh
cd docs
./migrate.py /path/to/QField-docs
pipenv run mkdocs serve
```

Preview: https://opengisch.github.io/qfield-doc-preview/en/

TODO:

- [ ] Check / fix integration with transifex
  - Supersmall German preview https://htmlpreview.github.io/?https://github.com/opengisch/QField/blob/gh-pages-test/de/index.html
  - Transifex test project https://www.transifex.com/opengisch/qfdocstest/dashboard/
- [ ] Cleanup leftover rst markup
- [ ] Integrate fancybox

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9669c9c8de38ecf34cc5dfffb85fb559b49624fe:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-95bd900da9d4290b0eb1ed4740b1a80adb9b2bd9-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-95bd900da9d4290b0eb1ed4740b1a80adb9b2bd9-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-95bd900da9d4290b0eb1ed4740b1a80adb9b2bd9-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-95bd900da9d4290b0eb1ed4740b1a80adb9b2bd9-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0785d2bc1f5ce318ca7f790b591df651d9c2b486:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-87fb84ee7b0a53bf451844a2fe1eae073d1ad92f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-87fb84ee7b0a53bf451844a2fe1eae073d1ad92f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-87fb84ee7b0a53bf451844a2fe1eae073d1ad92f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-87fb84ee7b0a53bf451844a2fe1eae073d1ad92f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 5fe3de8701d2461ef0cd36e9b555135f577fbca6:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-cc9b0ad59277db9d5f97204f987e786223e8679d-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-cc9b0ad59277db9d5f97204f987e786223e8679d-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-cc9b0ad59277db9d5f97204f987e786223e8679d-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-cc9b0ad59277db9d5f97204f987e786223e8679d-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 872b6bfa807c3e08b1b8a3df876b1907305231cf:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-287ab52d559a079ba56f9f39e85b4397abef0882-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-287ab52d559a079ba56f9f39e85b4397abef0882-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-287ab52d559a079ba56f9f39e85b4397abef0882-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-287ab52d559a079ba56f9f39e85b4397abef0882-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for c47c9f101e8e1fa287512a7236b4c3eca965cd04:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-330ad288cae02f98f9177fe7fe82dea8647fe565-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-330ad288cae02f98f9177fe7fe82dea8647fe565-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-330ad288cae02f98f9177fe7fe82dea8647fe565-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-330ad288cae02f98f9177fe7fe82dea8647fe565-x86.apk)

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+      - name: Build docs
+        run: |
+          pipenv run mkdocs build
+
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.4
+        with:
+          branch: gh-pages-test
```
Comment (m-kuhn) [inline .github/workflows/docs.yml:54]: TODO

File: docs/.tx/config [hunk 2]
```diff
@@ -0,0 +1,345 @@
+[main]
```
Comment (m-kuhn) [inline docs/.tx/config:1]: This file is generated and shouldn't be checked in

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 957f2c1d76213fc202efd4ce8ed50006ef8e3963:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 957f2c1d76213fc202efd4ce8ed50006ef8e3963:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6e6782eaad7ea73acb5b3391821a2a0727f653e7-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 3fa599361fd648e38b6752d75cfc64490b94628d:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e60e2154d2117d26b7943c2bb22186abee1f27ee-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e60e2154d2117d26b7943c2bb22186abee1f27ee-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e60e2154d2117d26b7943c2bb22186abee1f27ee-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e60e2154d2117d26b7943c2bb22186abee1f27ee-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 37c8e113095625daa498078387df12c0629e8728:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-28783f55eae05c1373539284ad091fe22bff421c-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-28783f55eae05c1373539284ad091fe22bff421c-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-28783f55eae05c1373539284ad091fe22bff421c-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-28783f55eae05c1373539284ad091fe22bff421c-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 18fa77cd987deb9239d8ce33602c84c0e0f60911:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bbd5bd7eb4b7dbae5370563042fe40d01e4d4342-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bbd5bd7eb4b7dbae5370563042fe40d01e4d4342-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bbd5bd7eb4b7dbae5370563042fe40d01e4d4342-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bbd5bd7eb4b7dbae5370563042fe40d01e4d4342-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f0bb50cd3fed7b30a7de7e3fc2b70087f47e6aae:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-a7e7b6f259f8569be5df049abb95805333f38db0-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-a7e7b6f259f8569be5df049abb95805333f38db0-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-a7e7b6f259f8569be5df049abb95805333f38db0-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-a7e7b6f259f8569be5df049abb95805333f38db0-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for cef7de3b197c4b28bc793a430c0a740e634540da:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e4086e6c33260fbbae4c365df4a77fd1eb1789a3-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e4086e6c33260fbbae4c365df4a77fd1eb1789a3-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e4086e6c33260fbbae4c365df4a77fd1eb1789a3-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e4086e6c33260fbbae4c365df4a77fd1eb1789a3-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 91fae467db593d8f4356edc09dbe84e18b2dcba9:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-0f7108517a10c6c188497fdd4d7164d7eb2e5c5f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-0f7108517a10c6c188497fdd4d7164d7eb2e5c5f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-0f7108517a10c6c188497fdd4d7164d7eb2e5c5f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-0f7108517a10c6c188497fdd4d7164d7eb2e5c5f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d0d44a042031979aced06e025e8851aae207ead6:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-844bbd06b193c42fb1362207cdf2c0b6e9738b94-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-844bbd06b193c42fb1362207cdf2c0b6e9738b94-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-844bbd06b193c42fb1362207cdf2c0b6e9738b94-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-844bbd06b193c42fb1362207cdf2c0b6e9738b94-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4de514b45a41cc801654f41001c451f64a6f7622:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-7d28ff5fb2b24d68bff78650f15e060e68a53c4b-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-7d28ff5fb2b24d68bff78650f15e060e68a53c4b-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-7d28ff5fb2b24d68bff78650f15e060e68a53c4b-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-7d28ff5fb2b24d68bff78650f15e060e68a53c4b-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 154e5960f6bc67a77b23a6ddb90289c354c7f2f8:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-36814ec60126b76542100ceb1e65948997e64034-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-36814ec60126b76542100ceb1e65948997e64034-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-36814ec60126b76542100ceb1e65948997e64034-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-36814ec60126b76542100ceb1e65948997e64034-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for de7fc91fcfb7fa28d535a0ff466f53122762a1af:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-5c3cba48078a985974c560199e1ae2174ba8f039-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-5c3cba48078a985974c560199e1ae2174ba8f039-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-5c3cba48078a985974c560199e1ae2174ba8f039-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-5c3cba48078a985974c560199e1ae2174ba8f039-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 683e7a3b794234b31f6135991c5ddd430758f07c:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-adce9c3ad3b19ea930ca3b63c87313b05f49ad8f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-adce9c3ad3b19ea930ca3b63c87313b05f49ad8f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-adce9c3ad3b19ea930ca3b63c87313b05f49ad8f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-adce9c3ad3b19ea930ca3b63c87313b05f49ad8f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 9e8d2a236ef2ecce77288672cf1ad2d431aa4a58:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e014709c21aa95964e7811c50b2ad2ff0437838f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e014709c21aa95964e7811c50b2ad2ff0437838f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e014709c21aa95964e7811c50b2ad2ff0437838f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e014709c21aa95964e7811c50b2ad2ff0437838f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 6b8e3453024364e50382e6bf969cfe342b667223:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6f61bff0796913b3c9a1e30fefd1f783a26a3c5a-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6f61bff0796913b3c9a1e30fefd1f783a26a3c5a-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6f61bff0796913b3c9a1e30fefd1f783a26a3c5a-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-6f61bff0796913b3c9a1e30fefd1f783a26a3c5a-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 54cf38e7341666f7d6a663608c5febdc85818e18:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bb676d2b160eede1871f1d3d826df3c630982daa-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bb676d2b160eede1871f1d3d826df3c630982daa-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bb676d2b160eede1871f1d3d826df3c630982daa-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-bb676d2b160eede1871f1d3d826df3c630982daa-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 8db892a76cd719f3ffaa271574b537bd69e41279:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-4ea54e1b8fcc866f57626f9bf7cf8e221908333f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-4ea54e1b8fcc866f57626f9bf7cf8e221908333f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-4ea54e1b8fcc866f57626f9bf7cf8e221908333f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-4ea54e1b8fcc866f57626f9bf7cf8e221908333f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 2d474e612e8c47c7547703e4c65710ba71e23953:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e28b2c8a780dee3fbe3a8970b55f9a13b71073bc-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e28b2c8a780dee3fbe3a8970b55f9a13b71073bc-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e28b2c8a780dee3fbe3a8970b55f9a13b71073bc-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-e28b2c8a780dee3fbe3a8970b55f9a13b71073bc-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 2102310e748829d770ce295fce997020736ea21f:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-ac1a03dcf001bfa9383decae63c6c0b2e52e08eb-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-ac1a03dcf001bfa9383decae63c6c0b2e52e08eb-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-ac1a03dcf001bfa9383decae63c6c0b2e52e08eb-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-ac1a03dcf001bfa9383decae63c6c0b2e52e08eb-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for c77f96d9e789a86f5ca044a182926ca9cfaf3774:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-2441d49c38e9faecd94ff14b9ebce3de49bf1d22-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-2441d49c38e9faecd94ff14b9ebce3de49bf1d22-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-2441d49c38e9faecd94ff14b9ebce3de49bf1d22-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2121-2441d49c38e9faecd94ff14b9ebce3de49bf1d22-x86.apk)

**Comment by m-kuhn** [comment]
@marioba I'll close this but not delete the branch

**Pull request closed without merging**